dropshots/src/main/java/com/dropbox/dropshots/Dropshots.kt

```diff
@@ -143,10 +144,8 @@ public class Dropshots : TestRule {
    * Writes the given screenshot to the external storage directory.
    */
   private fun writeImage(name: String, screenshot: Bitmap): String {
-    val externalStoragePath = System.getenv("EXTERNAL_STORAGE")!!
-    val screenFolder = File("$externalStoragePath/screenshots/${context.packageName}")
+    val screenFolder = "${Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS).absolutePath}//screenshots/${context.packageName}"
```

Comment:
@rharter this should get you started. I wasn't exactly sure if I have access to Android.os.Environment with the plugin. We would need to update the pull task as well. Happy to do it if just an import